src/beautifiers/index.coffee

```diff
@@ -140,20 +140,31 @@ module.exports = class Beautifiers extends EventEmitter
     ) or beautifiers[0]
     return beautifier
 
-  getLanguage : (grammar, filePath) ->
+  getExtension : (filePath) ->
+    if filePath
+      return path.extname(filePath).substr(1)
+
+  getLanguages : (grammar, filePath) ->
     # Get language
-    fileExtension = path.extname(filePath)
-    # Remove prefix "." (period) in fileExtension
-    fileExtension = fileExtension.substr(1)
-    languages = @languages.getLanguages({grammar, extension: fileExtension})
-    logger.verbose(languages, grammar, fileExtension)
-    # Check if unsupported language
-    if languages.length < 1
-      return null
+    fileExtension = @getExtension(filePath) 
+
+    if fileExtension
+      languages = @languages.getLanguages({grammar, extension: fileExtension})
+      logger.verbose(languages, grammar, fileExtension)
```

Comment:
Could you move this outside of the `if` statement? I'd like to see this in the logs, regardless of `fileExtension` value. I want to see in the logs if it is `undefined` 😉 .